src/merge.js

```diff
@@ -19,6 +19,9 @@ var _extend = require('./internal/_extend');
  *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': 40 });
  *      //=> { 'name': 'fred', 'age': 40 }
  *
+ *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': undefined });
+ *      //=> { 'name': 'fred', 'age': undefined }
```

Comment:
Perhaps [`R.dissoc`][1] is more appropriate in this case.


[1]: http://ramdajs.com/docs/#dissoc